lib/config/contributors.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const YAML = require('yaml').default;
+const fs = require('../deps/fsp.js');
+const path = require('path');
+
+const CONTRIBUTORS_CONFIG_LOCATION = path.resolve(__dirname, '..', '..', 'contributors', '_contributors.yaml');
+const CONTRIBUTORS_PICTURES_LOCATION = path.resolve(__dirname, '..', '..', 'content', 'images', 'contributors');
+
+let authorsConfig;
+
+async function loadContributorFromConfiguration(author) {
+    if (!authorsConfig) {
+        const contributorsFile = await fs.readFile(CONTRIBUTORS_CONFIG_LOCATION, 'utf8');
```

Comment:
2 space indents

Reply:
let's see if we can add some linting to enforce this.

Reply:
https://github.com/GoogleChrome/web.dev/issues/538

Reply:
Yeeee for writing code on my Mac and not on my linux system 🎉 😢 Updated my VSCode to 2 spaces and fixed indentation here.